tool/gravity/cli/status.go

```diff
@@ -94,7 +94,11 @@ func status(env *localenv.LocalEnvironment, printOptions printOptions) error {
 			env.Println("Failed to query Gravity cluster status. Running additional checks")
 		}
 
-		failed = checks.RunBasicChecks(ctx, nil)
+		localSite, err := clusterOperator.GetLocalSite(ctx)
+		if err != nil {
+			log.WithError(err).Warn("Failed to get local site.")
+		}
+		failed = checks.RunBasicChecks(ctx, nil, localSite.App.Manifest.OpenEBSEnabled())
```

Comment:
What will happen if the above call returns an error?

Reply:
And also, why is this called here, in the `gravity status` command?

Reply:
> And also, why is this called here, in the `gravity status` command?

In order to handle the case of a user installing iscsid on the host after the installation is complete. In this case the status command will show that there is a conflict with iscsid. I will remove it If we don't want to handle this scenario.

Reply:
> What will happen if the above call returns an error?

I will remove the OpenEBS related checks from here because it seems that we don't want to handle the case of a user installing iscsid on the host after the deployment is complete.